build.sbt

```diff
@@ -241,6 +254,8 @@ lazy val mediator = project
     Assets / WebKeys.packagePrefix := "public/",
     Runtime / managedClasspath += (Assets / packageBin).value,
   )
+  .settings(run / fork := true)
```

Comment:
Did you test if this works well with `reStart` sbt command?

Reply:
No Good Catch It wont work